Analyzer/plugins/Analyzer.cc

```diff
@@ -4421,10 +4505,10 @@ bool Analyzer::passPreselection(const reco::TrackRef track,
     if (GenBeta >= 0) {
       tuple->PostPreS_GenBeta->Fill(GenBeta, EventWeight_);
     }
-      
+
   }
- 
-  if (globalIas_ > 0.6 || Mass > 1000 || debug_ > 7 ) {
+
+  if ((globalIas_ > 0.6 || Mass > 1000) && debug_ > 7 ) {
```

Comment:
well this was not a bug... I want to see anything that has high mass or high ionization... even if the debug_ level is low... this doesnt happen very often anyway

Reply:
It doesn't happen a lot for background right? for signal, it seems to happen a lot. How about I change it to && debug_>0 ?

Reply:
Ah yes true, but then maybe we can have the protection that if SaveTree is 0 then do these printouts otherwise don't. What do you think?

Reply:
sounds good, i will change that